### PR TITLE
Split SparkSubmissionContentPanel from SparkSubmissionExDialog

### PR DESCRIPTION
**Pull request by djlan**
Split SparkSubmissionContentPanel from SparkSubmissionExDialog so the Spark submission UI can be reused

### CODE REVIEW

**Comment by vscjavaci** [comment]
Can one of the admins verify this patch?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/RemoteDebugSettingsEditor.java [hunk 1]
```diff
@@ -42,6 +58,17 @@ protected void applyEditorTo(@NotNull RemoteDebugRunConfiguration remoteDebugRun
     @NotNull
     @Override
     protected JComponent createEditor() {
-        return new JPanel();
+
+        Project project = myRunConfiguration.getProject();
+        List<IClusterDetail> cachedClusterDetails = ClusterManagerEx.getInstance().getClusterDetailsWithoutAsync(true, project);
```
Comment (voidfunction) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/RemoteDebugSettingsEditor.java]: >getClusterDetailsWithoutAsync [](start = 83, length = 29)

getClusterDetailsWithoutAsync is just getting all cached clusters. Is it on purpose?

**Comment by voidfunction** [comment]
:shipit:

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/RemoteDebugSettingsEditor.java (hunk 1 above)
Comment (djlan) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/RemoteDebugSettingsEditor.java]: It is not on purpose. However, we do want this dialog to pop up promptly. I will open an issue so we can get the no cluster bug fixed. Then this won't be an issue.

---
In reply to: [116899228](https://github.com/Microsoft/azure-tools-for-java/pull/447#discussion_r116899228) [](ancestors = 116899228)

**Pull request merged**